migrations/0012_auto_20200909_1547.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-09-09 15:47
+from __future__ import unicode_literals
+
+import core.file_system
+from django.db import migrations, models
+import django.db.models.deletion
+import plugins.books.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('books', '0011_auto_20190731_1456'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Category',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('slug', models.SlugField(max_length=255)),
+            ],
+            options={
+                'ordering': ('slug',),
+            },
+        ),
+        migrations.AlterField(
+            model_name='book',
+            name='cover',
+            field=models.FileField(blank=True, null=True, storage=core.file_system.JanewayFileSystemStorage(), upload_to=plugins.books.models.cover_images_upload_path),
```

Comment:
This field needs to declare a `storage` path ( `storage=JanewayFileSystemStorage('files/plugins/books'`) ) Otherwise the files would be dumped at the base path.

Reply:
It has `upload_to=plugins.books.models.cover_images_upload_path` which uploads to `media/cover_images` as expected.

Reply:
Ah fair enough, I thought we wanted plugin files to always land on /files/plugins but for a media file I guess it makes sense.